src/Application.js

```diff
@@ -175,13 +183,14 @@ export default class Application extends Dispatcher {
    * @return {Promise}
    */
   _init() {
-    this._setupConfig()
-    this._pluginInstances = {}
-    return this._loadPlugins().then(::this._boot).then(() => {
-      if (!this._appWatcher && !this.config.script && this.config.NODE_ENV != 'production') {
-        this.log.debug('Setting up App watcher')
-        this._appWatcher = new Watcher(this, this._getWatchPaths(), 'change', this._getAppIgnorePaths())
-      }
+    return this._setupConfig().then(() => {
+      this._pluginInstances = {}
```

Comment:
Not sure of sequencing – is it safe to defer initializing `_pluginInstances`?

Reply:
Yep